frappe/core/doctype/user_invitation/test_user_invitation.py

```diff
@@ -13,6 +13,7 @@
 )
 from frappe.core.doctype.user_invitation.user_invitation import mark_expired_invitations
 from frappe.tests import IntegrationTestCase
+from frappe.tests.test_query import convert_identifier_quotes
```

Comment:
Use `self.assertQueryEqual`, if required, fix that function so it compares queries without checking for DB specific identifiers.